### DISCUSSION TITLE
GENROU generates additional power during secondary frequency regulation

### DISCUSSION THREAD

**Discussion by Cmt111** [CURENT/andes#486] [Q&A]
Hello Dr. Cui

When simulating secondary frequency regulation, I encountered a problem. The system used is
ss = andes.load(andes.get_case("ieee14/ieee14_full.xlsx"), setup=False, default_config=True) .
How to control GENROU increase a specific power at a certain moment? Which variables should be modified and how to calculate them?

Could you please address this confusion.
Regards

**Comment by jinningwang** [comment]
Check this example, https://docs.andes.app/en/latest/_examples/ex8.html?highlight=Paux#step-4-apply-the-auxiliary-power-setpoints-to-tgov1-paux0-v
